app/scripts/components/misc/t.jsx

```diff
@@ -43,6 +54,13 @@ const T = React.createClass({
     if (typeof translated === 'undefined') {
       warn('missing translation for key', this.props.k);
       translated = this.props.k;
+    } else if (typeof translated === 'function') {
+      if (!(this.propTypes.i instanceof Array)) {
+        warn('missing expected array for interpolating', this.props.k, 'got:', this.propTypes.i);
+        translated = translated();
+      } else {
+        translated = translated(...this.props.i);
```

Comment:
it's an ugly way of doing variable interpolation. From line 38 above:
```js
'error.api.ckan.unknown-field-type': i => `The CKAN data server returned data of an unknown type '${i[0]}' for field '${i[1]}'`,
```
some keys are static strings, some are functions that take an array to interpolate. This `if` checks if it's a function, and executes it if so, saving its result back to `translated`.